services.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: Services for Flutter
+permalink: /services/
+---
+
+Flutter helps you access system services and 3rd party SDKs.
+For example, you can open files, connect to network sockets,
+activate the keyboard, talk to Firebase, and more. Flutter
+ships with some built-in services. In the future, we plan to make
+it possible for developers to build their own services.
+
+This document lists the existing services that either
+ship with Flutter, or are available as an add-on.
+We expect this list to grow.
+If there's a service that you need for your app, please let
+us know by [filing an issue][newissue]. Thanks!
+
+_Note: we're still building out docs and examples. If you have
+any questions, don't hesitate to [contact us](/contact/).
+Doc contributions are also welcome. :)_
+
+## Out of the box
+
+The following services are available out of the box.
+
+### Cross platform
+
+* Keyboard
+  * docs
+  * example
+* Audio
+  * docs
+  * example
+* Files
+  * docs
+  * example
+* Network
+  * docs
+  * example
+* Sensors
+  * docs
+  * example
+* Haptic feedback
+  * docs
+  * example
+
+### Android only
+
+* Activity
+  * docs
+  * example
+* Intent
+  * docs
+  * example
+
+## Add-ons
+
+The following services are available for Flutter, but
+require an additional installation step.
+
+### Cross platform
+
+* Firebase
+  * [docs](https://github.com/flutter/firebase-dart/blob/master/README.md)
+  * [example](https://github.com/flutter/firechat-flutter)
+* Google Cloud Messaging (GCM)
```

Comment:
BTW, ios support for GCM is not implemented. Maybe put this under android-only for now.